classes/cve-check.bbclass

```diff
@@ -64,15 +64,28 @@ python do_cve_check () {
             if len(cip_kernel_sec_affected_cves) > 0:
                 tmp = cip_kernel_sec_affected_cves.split(" ")
                 for cve in tmp:
-                    if not cve in unpatched:
+                    # Do not override poky's cve-check result when cve is in patched list.
+                    if not cve in unpatched and not cve in patched:
                         unpatched.append(cve)
 
+                unpatched = sorted(unpatched)
+
+            cip_kernel_sec_fixed_cves = d.getVar("CIP_KERNEL_SEC_FIXED_CVES")
+            if len(cip_kernel_sec_fixed_cves) > 0:
+                tmp = cip_kernel_sec_fixed_cves.split(" ")
+                for cve in tmp:
+                    if not cve in patched or cve in unpatched:
+                        # in case poky's cve-check thought this cve is unpatched but cip-kernel-sec said it's fixed
+                        patched.append(cve)
```

Comment:
If a cve is on the "unpatched" list, is it OK to leave the cve on that list?

Reply:
@hiraku-wfs 

The only condition for cip-kernel-sec to determine that a bug has been fixed is if the commit that fixed it is registered in fixed-by and that commit is included in the kernel being checked, so there is no problem in using the results reported by cip-kernel-sec that a bug has been fixed.